test/platforms.test.js

```diff
@@ -15,39 +27,89 @@ describe("platforms.js", function () {
     // See the following for a list of all possible `platform` values:
     // https://nodejs.org/api/os.html#osplatform
```

Comment:
This comment doesn't really make sense here anymore. It should probably move to [the `common.js` file](https://github.com/ericcornelissen/shescape/pull/192/files#diff-5c1ae2b5e56dedb8288bbde987415e6c63f6805f2e1d07e8c04cbf4e1b5fcc4bR9).